AdobeStockImageAdminUi/view/adminhtml/ui_component/adobe_stock_images_listing.xml

```diff
@@ -35,7 +35,7 @@
     </dataSource>
     <listingToolbar name="listing_top" template="Magento_AdobeStockImageAdminUi/grid/toolbar">
         <settings>
-            <sticky>true</sticky>
+            <sticky>false</sticky>
```

Comment:
False appears to be the default value, I believe we can simply remove settings node

Reply:
@sivaschenko ... you are right. I have removed the settings node from the file.

Reply:
some modules still use this value. Remove settings value seem not fix actually problem. In case we need use true sticky

Reply:
Customer, Cms still use sticky in listing grid

Reply:
@mrtuvn we don't need a sticky header for Adobe Stock Integration. The root cause for this issue should be addressed in magento2 repository, here is a corresponding issue https://github.com/magento/magento2/issues/26147